diff --git a/src/auditwheel/policy.rs b/src/auditwheel/policy.rs
index dc52e25..bfb1114 100644
--- a/src/auditwheel/policy.rs
+++ b/src/auditwheel/policy.rs
@@ -104,6 +104,7 @@ impl Policy {
                 Arch::X86 => "libc.musl-x86.so.1",
                 Arch::X86_64 => "libc.musl-x86_64.so.1",
                 Arch::S390X => "libc.musl-s390x.so.1",
+                Arch::LoongArch64 => "libc.musl-loongarch64.so.1",
                 _ => "",
             };
             if !new_soname.is_empty() {
